datafusion/sqllogictest/test_files/select.slt

```diff
@@ -483,6 +483,17 @@ CREATE TABLE foo2(c1 double, c2 double) AS VALUES
 (2, 5),
 (3, 6);
 
+statement ok
+SELECT COALESCE(column1, column2) FROM VALUES (null, 1.2);
```

Comment:
Maybe add `SELECT arrow_typeof(COALESCE(column1, column2)) FROM VALUES (null, 1.2);` to show that the final coerced type?